### ISSUE TITLE
Nemo crashes if you want to move a link over tabs

### ISSUE THREAD

**Issue by Toorero** [linuxmint/nemo#2746]
```
 * Nemo version 4.8.5
 * Is issue with desktop or windowed nemo? Not sure  what this is about
 * Arch
 * Graphics hardware *and* driver used: AMD 5700XT, amdgpu
 * 64 bit
 ```

**Issue**
Nemo crashes if trying to link a folder into a different tab.

**Steps to reproduce**
1. Open two taps in Nemo.
2. Drag a folder with alt button pressed down into **any** of the open tabs.
3. See Nemo crash.

**Expected behaviour**
No crash pls

**Other information**
The linking is still performed I guess before Nemo crashes.

**Comment by Jeremy7701** [comment]
On Nemo version 4.8.6 (current) I tried this in a nemo window on LMDE4.

Couldn't reproduce the fault.
I was given the option to copy, move or create a link.
So I chose create a symbolic link in the second tab -> first tab.

**Comment by Toorero** [comment]
Well ok perhaps the issue is fixed in 4.8.6 then

**Comment by Toorero** [comment]
Ok I finally was able to adopt the pkgbuild file for nemo-git because weirdly 4.8.6 isn't merged into master and I can confirm this issue still exists for me in 4.8.6

**Comment by icarter09** [comment]
Nemo v4.8.5
Mint 20.1 64bit

I'm not able to reproduce this issue. I'm experience the same as @Jeremy7701.

**Comment by Toorero** [comment]
Yeah weird. Perhaps it's a library dependency issue because it clearly exists on Arch Linux. Can someone tell me how I would debug this issue?

**Comment by Jeremy7701** [comment]
Does a twin-pane approach work for you?
Shortcut is F3.
Realise its a workaround.

**Comment by Toorero** [comment]
In deed the "twin-pane" approach works for me.

**Comment by LaurentOngaro** [comment]
similar issue for me

**Env**
nemo version 5.0.3
gnome shell version 40.3
using Wayland
OS is not linux mint but manjaro 21.1.0

**Issue**
Nemo crashes if trying to link a folder into a different tab.

**Steps to reproduce**

open 2 panels using F3
drag a folder from left to right panels maintaining the ALT key pressed
choose an option in the menu (copy/move/link)

**Result**
nemo crashes

**NOTES**
The selected action is DONE despite the crash

The following errors appear if nemo is launched via a terminal:

```
Gdk-Message: 09:38:41.081: Window 0x5591b91416a0 is a temporary window without parent, application will not be able to position it on screen.

(nemo:19178): Gdk-CRITICAL **: 09:38:41.085: gdk_wayland_window_handle_configure_popup: assertion 'impl->transient_for' failed
Gdk-Message: 09:38:45.000: Error flushing display: Relais brisé (pipe)
```

**Comment by Toorero** [comment]
I can't reproduce the error this way. [For me, the F3 panels work fine](https://github.com/linuxmint/nemo/issues/2746#issuecomment-846610636).

**Comment by Toorero** [comment]
But the other error still persists in 5.0.3 for me
